main.tf

```diff
@@ -141,6 +141,25 @@ resource "google_service_account_iam_binding" "workload_identity_binding" {
   depends_on = [google_iam_workload_identity_pool.create_wip]
 }
 
+resource "google_project_iam_custom_role" "role_for_uptycs" {
```

Comment:
I think it creates a role in specific project not in organization and also below we are giving permissions at project level. Can you check